src/controllers/userController.js

```diff
@@ -59,5 +61,38 @@ const signUp = async (req, res) => {
   });
 };
 
+/**
+   * user login controller
+   * @param {Object} req - server request
+   * @param {Object} res - server response
+   * @return {Object} - custom response
+   */
+const login = async (req, res) => {
+  const {
+    email, password
+  } = req.body;
+
+  const authErrorMessage = [{ field: 'email or password', message: 'email or password is incorrect' }];
+
+  const foundUser = await User.findOne({ where: { email } });
+
+  if (foundUser) {
+    const correctPassword = authHelper.comparePassword(password, foundUser.password);
+    if (correctPassword) {
```

Comment:
can we avoid having nested if statements?

Reply:
do something like 
`let correctPassword; then you can do your checks`

Reply:
Okay, on it!